subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
                                 </ul>
                             </li>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#java-web-projects" aria-expanded="false" aria-controls="java-web-projects">Java Web Projects</a>
-                                <ul id="java-web-projects">
-                                    <li><a href="/userguide/web_project_tutorial.html">Web Application Quickstart</a></li>
-                                    <li><a href="/userguide/ear_plugin.html">EAR Plugin</a></li>
-                                    <li><a href="/userguide/play_plugin.html">Play Plugin</a></li>
-                                    <li><a href="/userguide/war_plugin.html">WAR Plugin</a></li>
-                                </ul>
-                            </li>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#scala-projects" aria-expanded="false" aria-controls="scala-projects">Scala Projects</a>
-                                <ul id="scala-projects">
-                                    <li><a href="/userguide/scala_plugin.html">Scala Plugin</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#advanced-techniques" aria-expanded="false" aria-controls="advanced-techniques">Advanced techniques</a>
+                                <ul id="authoring-build-scripts">
+                                    <li><a href="/userguide/lazy_configuration.html">Configuring tasks lazily</a></li>
+                                    <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing parallel tasks</a></li>
+                                    <li><a href="/userguide/test_kit.html">Testing your build with testkit</a></li>
+                                    <li><a href="/userguide/ant.html">Using Ant from Gradle</a></li>
                                 </ul>
                             </li>
-                        </ul>
-
-                        <h3 id="integrating-gradle">Integrating Gradle</h3>
-                        <ul>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#ci-integration" aria-expanded="false" aria-controls="ci-integration">Continuous Integration</a>
-                                <ul id="ci-integration">
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-jenkins">Using Gradle with Jenkins</a></li>
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-teamcity">Using Gradle with TeamCity</a></li>
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-travisci">Using Gradle with Travis CI</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#best-practices" aria-expanded="false" aria-controls="best-practices">Best practices</a>
```

Comment:
❓ Don't you think "Best Practices" should be prioritized over advanced techniques and perhaps other sections?